ImagePicker/ImagePickerController.swift

```diff
@@ -199,67 +220,68 @@ open class ImagePickerController : UIViewController {
         }
         return instanceAppearanceProxy!
     }
-    
+
     ///
     /// A collection view that is used for displaying content.
     ///
     public var collectionView: UICollectionView! {
         return imagePickerView.collectionView
     }
-    
+
     // MARK: Private Methods
-    
-    private var collectionViewCoordinator: CollectionViewUpdatesCoordinator!
-    
+
+    fileprivate var collectionViewCoordinator: CollectionViewUpdatesCoordinator!
+
     fileprivate var imagePickerView: ImagePickerView! {
-        return view as! ImagePickerView
```

Comment:
this doesn't affects functionality. if kept as fatal please specify error message if possible